src/baseclient.cc

```
@@ -2049,4 +2050,488 @@ UploadPartCopyResponse BaseClient::UploadPartCopy(UploadPartCopyArgs args) {
+                    });
+}
+
+std::future<ListObjectsResponse> BaseClient::ListObjectVersionsAsync(
+    ListObjectVersionsArgs args) {
+  return std::async(std::launch::async,
+                    [this, args = std::move(args)]() mutable {
+                      return ListObjectVersions(std::move(args));
+                    });
+}
+
+std::future<MakeBucketResponse> BaseClient::MakeBucketAsync(
```

Cppcheck (2.21.0)

[style] The function 'MakeBucketAsync' is never used.

(unusedFunction)